gutterball/pom.xml

```diff
@@ -821,6 +822,14 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>org.candlepin</groupId>
+      <artifactId>candlepin-common</artifactId>
+      <version>2.0.0</version>
+      <classifier>complete</classifier>
```

Comment:
No, the `pom` plugin will take care of that.  After every update to common, you will need to run `buildr pom` but we should be doing that frequently anyway.